src/blocks/modal.css

```diff
@@ -145,6 +145,7 @@
   font-weight: 500;
   font-size: 32px;
   line-height: 1;
+  font-family: Poppins;
```

Comment:
It's important to always provide **fallback** and **generic** font name for `font-family`, like that: 

```css

font-family: Poppins, Arial, Helvetica, sans-serif;

```

https://developer.mozilla.org/en/docs/Web/CSS/font-family

 

Please, make this correction for **all the places** where you use `font-family`